core/src/main/scala/com/rallyhealth/vapors/core/dsl/factfilter/Dsl.scala

```diff
@@ -78,20 +79,21 @@ private[dsl] class Dsl extends Types with Syntax {
 
   def alwaysFalse[T]: CondExp[T] = liftCondExp(ExpAlg.Pure("False", _ => false))
 
-  def alwaysMatch: TerminalFactsExp[Any] = liftTermExp(ExpAlg.Pure("AlwaysMatch", FactsMatch(_)))
+  def alwaysMatch: AnyTerminalFactsExp = liftAnyTermExp(ExpAlg.Pure("AlwaysMatch", FactsMatch(_)))
 
-  def alwaysEmpty: TerminalFactsExp[Any] = liftTermExp(ExpAlg.Pure("AlwaysEmpty", _ => NoFactsMatch()))
+  def alwaysEmpty: AnyTerminalFactsExp = liftAnyTermExp(ExpAlg.Pure("AlwaysEmpty", _ => NoFactsMatch()))
 
   // Helper methods for building conditional expressions that terminate in a functor to boolean
   private def True[X]: X => Boolean = _ => true
   private def False[X]: X => Boolean = _ => false
 
-  /** @see [[TerminalFactsExp]] */
-  private def liftTermExp[T](value: ExpAlg[Facts[T], ResultSet[T]]): TerminalFactsExp[T] = FreeApplicative.lift(value)
+  /** @see [[FactsExp]] */
+  private def liftAnyTermExp(value: ExpAlg[AbstractFacts, AbstractResultSet]): AnyTerminalFactsExp =
+    FreeApplicative.lift(value)
 
   /** @see [[CondExp]] */
   private def liftCondExp[X](value: ExpAlg[X, Boolean]): CondExp[X] = FreeApplicative.lift(value)
 
-  /** @see [[AnyExp]] */
-  private def liftAnyExp[X, A](value: ExpAlg[X, A]): AnyExp[X, A] = FreeApplicative.lift(value)
+  /** @see [[Exp]] */
+  private def liftExp[X, A](value: ExpAlg[X, A]): Exp[X, A] = FreeApplicative.lift(value)
```

Comment:
value or "expression"

Reply:
Good idea, I'll rename these to expression.